hircluster.c

```diff
@@ -1848,6 +1860,34 @@ int redisClusterSetOptionConnectNonBlock(redisClusterContext *cc) {
     return REDIS_OK;
 }
 
+/**
+ * Configure a username used during authentication, see
+ * the Redis AUTH command.
+ * Disabled by default. Can be disabled again by providing an
+ * empty string or a null pointer.
```

Comment:
This differs from how password is handled. Maybe we should change password?

Reply:
Yes, I think so too. I started to fix it in a similar way in a separate PR.